docs/source/implementations/feature_importance.rst

```diff
@@ -58,16 +59,22 @@ An example showing how to use various feature importance functions:
 
   oos_score = ml_cross_val_score(clf, X_train, y_train, cv_gen=cv_gen, sample_weight_train=None, scoring=accuracy_score).mean()
 
+  feature_clusters = get_feature_clusters(X_train, dependence_metric='information_variation', distance_metric='angular',
+                                          linkage_method='single', n_clusters=None)
+
   mdi_feature_imp = mean_decrease_impurity(clf, X_train.columns)
   mda_feature_imp = mean_decrease_accuracy(clf, X_train, y_train, cv_gen, scoring=log_loss)
   sfi_feature_imp = single_feature_importance(clf, X_train, y_train, cv_gen, scoring=accuracy_score)
+  cfi_feature_imp = mean_decrease_accuracy(clf, X_train, y_train, cv_gen, clustered_subsets=feature_clusters, scoring=log_loss)
 
   plot_feature_importance(mdi_feat_imp, oob_score=clf.oob_score_, oos_score=oos_score,
                                 save_fig=True, output_path='mdi_feat_imp.png')
   plot_feature_importance(mda_feat_imp, oob_score=clf.oob_score_, oos_score=oos_score,
                                 save_fig=True, output_path='mda_feat_imp.png')
   plot_feature_importance(sfi_feat_imp, oob_score=clf.oob_score_, oos_score=oos_score,
                                 save_fig=True, output_path='sfi_feat_imp.png')
+  plot_feature_importance(cfi_feat_imp, oob_score=clf.oob_score_, oos_score=oos_score,
```

Comment:
Where is the plot for this image?

Reply:
I asked @proskurin for the dataset for the plot. He said just to show the implementation of CFI and leave the plot.

Reply:
I think its very important to add the plot, else it breaks the style of the document. People will say, oh you have the other 3 but not this 4th one? :)

Reply:
If you want I can do it with sklearn generated features.

Reply:
Yea that could work. Maybe we should update the other 3 to work with the same gen data set from sklearn?

@proskurin please chime in with your thoughts.

Reply:
I have created the plots for both Clustered MDI and MDA.